### DISCUSSION TITLE
Error: Found tag id where item is expected

### DISCUSSION THREAD

**Discussion by shajby1** [distriqt/ANE-IronSource#48] [Q&A]
Hi,

while compiling in Animate with Air 33.1.1.821 I m getting following errors:

![error](https://user-images.githubusercontent.com/5220110/233045222-0e6cae5c-1e97-47b2-b491-c3ae9f66f650.png)

Any help appreciated.

Thank you!

**Comment by marchbold** [comment]
Hi, You need to ensure you are using the new build method. Make sure you have provided a valid Android sdk path to your air configuration. 

Check the AIR SDK Manager if you need help setting this. Easiest way to install the android sdk is to install android studio.

**Comment by shajby1** [comment]
Hi, thanks for info. And yes this was caused because I was trying to export APK and not AAB format. When using AAB it works fine, thanks!
